cvat-ui/src/reducers/models-reducer.ts

```diff
@@ -47,6 +74,52 @@ export default function (state = defaultState, action: ModelsActions | AuthActio
                 fetching: false,
             };
         }
+        case ModelsActionTypes.CREATE_MODEL: {
+            return {
+                ...state,
+                fetching: true,
+            };
+        }
+        case ModelsActionTypes.CREATE_MODEL_FAILED: {
+            return {
+                ...state,
+                fetching: false,
+            };
+        }
+        case ModelsActionTypes.CREATE_MODEL_SUCCESS: {
+            const mutual = {
+                ...state,
+                fetching: false,
+            };
+            if ([ModelType.INTERACTOR].includes(action.payload.model.type)) {
```

Comment:
Fell like all this code can be reduced, like:

```
return {
 ...mutual,
[`${action.payload.model.type}s`]: [...`${action.payload.model.type}s`, action.payload.model]  
}
```

But this DOES NOT work for reid. Just a kind of idea to think about it.